src/components/LoginPanel.tsx

```diff
@@ -45,29 +56,32 @@ const LoginPanel: React.FC<{}> = () => {
 
     return (
         <MiddleDiv data-loaded={loaded} id="login-panel">
-            <Auth
-                supabaseClient={supabase}
-                providers={[]}
-                appearance={{
-                    theme: ThemeSupa,
-                    variables: {
-                        default: {
-                            colors: {
-                                inputText: theme.main.foreground[2],
-                                inputBackground: theme.main.background[2],
-                                inputBorder: theme.main.background[2],
-                                inputLabelText: theme.main.foreground[0],
-                                anchorTextColor: theme.main.lighterForeground[0],
-                                brand: theme.primary.background[2],
-                                brandAccent: theme.primary.background[2],
-                                brandButtonText: theme.primary.foreground[2],
-                                messageTextDanger: theme.error,
+            <Title>Login</Title>
+            <NoSsr>
+                <Auth
+                    supabaseClient={supabase}
+                    providers={[]}
+                    appearance={{
+                        theme: ThemeSupa,
+                        variables: {
+                            default: {
+                                colors: {
+                                    inputText: theme.main.foreground[2],
+                                    inputBackground: theme.main.background[2],
+                                    inputBorder: theme.main.background[2],
+                                    inputLabelText: theme.main.foreground[0],
+                                    anchorTextColor: theme.main.lighterForeground[0],
+                                    brand: theme.primary.background[2],
+                                    brandAccent: theme.primary.background[2],
+                                    brandButtonText: theme.primary.foreground[2],
+                                    messageTextDanger: theme.error,
+                                },
                             },
                         },
-                    },
-                }}
-                redirectTo="http://localhost:3000/auth/callback"
-            />
+                    }}
+                    redirectTo="http://localhost:3000/auth/callback"
```

Comment:
Is this `redirectTo` going to be valid in a production build?

Reply:
Dont think so but this will have to be done in a separate ticket.

Reply:
use `window.location.origin`